androidApp/src/test/kotlin/dev/johnoreilly/confetti/SpeakerDetailsScTest.kt

```diff
@@ -0,0 +1,44 @@
+package dev.johnoreilly.confetti
+
+import androidx.compose.ui.test.onNodeWithText
+import dev.johnoreilly.confetti.fragment.SpeakerDetails
+import dev.johnoreilly.confetti.speakerdetails.SpeakerDetailsView
+import dev.johnoreilly.confetti.ui.ConfettiTheme
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Test
+
+
+class SpeakerDetailsScTest : BaseScreenshotTest(false, a11yEnabled = true) {
```

Comment:
Should we put a theme background in so it's not transparent background.

Reply:
I think that would be nice! Will add that!😊

Reply:
Or do I have to set an explicit background?

Reply:
I suspect you'll have to set one. I think we are seeing that the mobile screens are transparent, so it must be falling back to a theme appropriate background.

Reply:
But I think taking the confetti theme background will work. It just leaves it technically possible that they differ in some cases.

Reply:
I don't think it's technically correct, since you might have a device theme that differs, but I think it's good enough to start.

Reply:
Aha, yeah alright. Let's keep it for a start then :)